kuryr_kubernetes/controller/drivers/vif_pool.py

```diff
@@ -202,10 +202,10 @@ def request_vif(self, pod, project_id, subnets, security_groups):
         try:
             return self._get_port_from_pool(pool_key, pod, subnets,
                                             tuple(sorted(security_groups)))
-        except exceptions.ResourceNotReady:
+        except exceptions.VIFPoolNotReady:
```

Comment:
If we don't raise a resourceNotReady here will the Pod event ever be retried?
Seems the VIFPoolNotReady exception is not being retried on the retry handler.

Reply:
probably it will by the reconciliation loop, right?

Reply:
@dulek can you confirm this one? thanks

Reply:
@MaysaMacedo: It's not explicitly listed, but `ResourceNotReady` is and `VIFPoolNotReady` is based on it:

```
class VIFPoolNotReady(ResourceNotReady):
```